### ISSUE TITLE
3.3.7: Ensuring copy-paste is not blocked

### ISSUE THREAD

**Issue by benja11y** [w3c/wcag#1878]
The "Intent" section of the non-normative Understanding doc mentions that copy-paste must not be blocked, but this is not in the text of the SC. We feel it would need to be in order to ensure that it's a requirement.

This issue has been raised on behalf of [TPGi](https://www.tpgi.com), based on an internal consultation process. [An article containing all of our comments](https://www.tpgi.com/wcag-22-working-draft-feedback/) can be found on our blog, along with an introduction and acknowledgments.

**Comment by alastc** [comment]
In the context of the whole SC, copy-paste must not be blocked if you are using a username/password and there is not alternative mechanism.

You could block copy-paste if you provided another mechanism to login that didn't require a cognitive function-test.

I assume you are referring to this bit:

> Blocking people or programs from pasting into authentication fields, or using a different format between the copied text and the input field, would force the user to transcribe information and therefore fail this criterion. 

Perhaps we should add "unless another method is available" to the end?

**Comment by alastc** [comment]
Hi @benja11y, does the response above make sense? Would this close off the issue for you?
